src/pages-and-resources/discussions/apps/shared/BlackoutDatesField.jsx

```diff
@@ -4,11 +4,14 @@ import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Form } from '@edx/paragon';
 import messages from './messages';
 
+export const blackoutDatesRegex = /^\[(\[("[0-9]{4}-(0[1-9]|1[0-2])-[0-3][0-9](T([0-1][0-9]|2[0-3]):([0-5][0-9])){0,1}"),("[0-9]{4}-(0[1-9]|1[0-2])-[0-3][0-9](T([0-1][0-9]|2[0-3]):([0-5][0-9])){0,1}")\](,){0,1})+\]$/;
```

Comment:
I'd suggest adding a comment summarizing the intent here, so future devs/support understand the intent.